thorlcr/activities/indexread/thindexreadslave.cpp

```diff
@@ -806,8 +827,16 @@ class CIndexReadSlaveActivity : public CIndexReadSlaveBase
                                     callback.finishedRow();
                                     return ret.finalizeRowClear(sz);
                                 }
+                                else
+                                {
+                                    ++postFiltered;
+                                }
```

Comment:
as per other trivial/stlye: comment - to be consistent with other code, don't normally put single statement in a curly block.